src/runtime/helpers.js

```diff
@@ -157,6 +158,8 @@ var helpers = {
                         r[removeDashes(key)] = attrs[key];
                         return r;
                     }, {});
+                } else if (attrs == null) {
+                    attrs = {};
```

Comment:
This was done because previously it was possible to have undefined input passed to a renderer with the dynamic tag. I believe this was an error.